### DISCUSSION TITLE
Running on Mac using Terminal, VS for Mac, or Vs code (MacOS)

### DISCUSSION THREAD

**Discussion by edys** [machine/machine.specifications#499] [Q&A] [answered]
Hey guys, I love this framework!

I have tried unsuccessfully in the past to run this on macs

I'm trying again. :)

Assuming this can be done, Are there any hints or tuts on how to set this up?

Thanks in advance

Edys

**Comment by robertcoltheart** [comment, marked as answer]
Mspec integrates with `dotnet test`, so as long as you have a test project configure correctly, and dotnet is install on Mac, you should be fine. There are plenty of examples on Stackoverflow for how to setup a test project, and the readme contains information on how to install Mspec.

**Reply by edys** [reply]
Cool, thank you,  will try tomorrow.
